components/compliance-service/cmd/compliance-service/cmd/run.go

```diff
@@ -34,6 +34,7 @@ var conf = config.Compliance{
 		TmpDir:              os.Getenv("TMPDIR"),
 		ResultMessageLimit:  10000,
 		ControlResultsLimit: 50,
+		RunTimeLimit:        1.0,
```

Comment:
should we have a comment on this to document the unit of time?  min? seconds?

Reply:
Good point, will put a comment and will mention it in the runFlags description.